transcriptic/cli.py

```diff
@@ -628,3 +628,66 @@ def login_cmd(ctx, api_root=None, analytics=True, rsa_key=None):
 def format_cmd(manifest):
     """Check Autoprotocol format of manifest.json."""
     commands.format(manifest)
+
+
+@cli.command("exec")
```

Comment:
nit: we probably want to limit this by feature group. See @cli.command("submit", ...)

Reply:
discussed offline